code/server/routes/index.js

```diff
@@ -28,6 +28,36 @@ router.get("/matches", function(req, res) {
 	});
 });
 
+router.get("/leaderBoard", function(req,res) {
```

Comment:
change "/leaderBoard" to "/leaders".  Didn't work on my computer until I did so.